supportedBackends/aws/src/main/resources/ecs-proxy/proxy

```diff
@@ -94,7 +94,13 @@ copyfile() {
     if [[ "$localname" == glob-*/"*" ]]; then
       awscp sync "${localdirectory}/${name}" "${s3location}"
     else
-      awscp cp "${localdirectory}/${localname}" "${s3location}"
+      localfilepath="${localdirectory}/${localname}"
+      if [ -e  "$localfilepath" ] ; then
+        awscp cp "${localfilepath}" "${s3location}"
+      else
+        echo "No such local file: ${localfilepath}"
```

Comment:
Pipe this to stderr?

Reply:
we chatted via slack earlier but for posterity - doing so won't accomplish anything (and **potentially** could have a deleterious effect) due to how/why this script is run and other candidates for stderr'ing are also going to stdout. I can explore that particular set of belts & suspenders when i return